app/src/Users.vue

```diff
@@ -28,8 +28,34 @@
         <p>You can't undo this action.</p>
       </ConfirmationDialog>
     </div>
+    <div class="search">
+      <TextInput
+        id="search"
+        v-model="searchPhrase"
+        pattern="^.{0,100}$"
+        title="Your search phrase is too long!"
```

Comment:
What is the purpose of the `title`?

Reply:
Removed it in the recent commit. Generally it is used to explain why the input is erroneous (why it does not falling with the specified pattern), but in this case it is not displayable, so I'll remove it.